models/book.js

```diff
@@ -0,0 +1,37 @@
+import mongoose from 'mongoose'
+
+const Schema = mongoose.Schema
+
+const BookSchema = new Schema({
+  bookID: {
+    type: Number
+  },
+  title: {
+    type: String
+  },
+  authors: {
+    type: String
+  },
+  average_rating: {
+    type: Number
+  },
+  isbn13: {
+    type: Number,
+    min: 9780000000000,
+    max: 9999999999999
+  },
+  language_code: {
+    type: String
+  },
+  num_pages: {
+    type: Number
+  },
+  ratings_count: {
+    type: Number
+  },
+  text_reviews_count: {
+    type: Number
+  },
```

Comment:
It will be safer for you if you add required: true to some fields. this will prevent from creating empty entries in the database ;)

Good that you added range for ISBN number :) 👍 

text_reviews_count is your last field, so you shouldn't add coma after it in line 34

Reply:
Ah, makes sense! Added it to some of them :)

Removed the comma :)